### PR TITLE
Feature/four 11534: Process Launchpad - Process Title - Description Add Ellipsis

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
When a process is open whe have the id 
`processes/{process}`

1. Add the title of the process
2. Add the description of the process
3. Add a ellipsis to show the same compoment from process list (located to the right of the process title)
image-20231128-170348

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-11534

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:next
ci:deploy

### CODE REVIEW

File: resources/js/processes-catalogue/components/ProcessMap.vue [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  <div id="processMap">
+    <div id="processData">
+      <div
+        id="header"
+        class="d-flex justify-content-between mb-3"
+      >
+        <h4 class="d-flex align-items-center">
+          <i
+            class="fas fa-arrow-circle-left text-secondary mr-2"
+            style="font-size: 32px"
```
Comment (fagubla) [inline resources/js/processes-catalogue/components/ProcessMap.vue:11]: create a new class this custom style

File: resources/js/processes-catalogue/components/ProcessMap.vue [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+          />
+          {{ process.name }}
+        </h4>
+        <span
+          class="border bg-white rounded-circle d-flex align-items-center p-0"
+          style="border-color: #CDDDEE;"
```
Comment (fagubla) [inline resources/js/processes-catalogue/components/ProcessMap.vue:17]: create a new class this custom style

File: resources/js/processes-catalogue/components/ProcessMap.vue [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    />
+  </div>
+</template>
+
+<script>
+import EllipsisMenu from "../../components/shared/EllipsisMenu.vue";
+import CreateTemplateModal from "../../components/templates/CreateTemplateModal.vue";
+import CreatePmBlockModal from "../../components/pm-blocks/CreatePmBlockModal.vue";
+import AddToProjectModal from "../../components/shared/AddToProjectModal.vue";
+
```
Comment (fagubla) [inline resources/js/processes-catalogue/components/ProcessMap.vue:63]: remove this blank line

**Review by fagubla — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-793eda3ccb.eng.processmaker.net

**Pull request merged**